test/e2e/log_size_monitoring.go

```diff
@@ -75,8 +75,34 @@ type LogsSizeVerifier struct {
 	workers       []*LogSizeGatherer
 }
 
+type LogsSizeDataSummary map[string]map[string][]TimestampedSize
+
+func (s *LogsSizeDataSummary) PrintHumanReadable() string {
+	buf := &bytes.Buffer{}
```

Comment:
it seems that you removed lock from here.
So, how do you guarantee that it will not change during printing?